_posts/2017/2017-06-27-why-a-techwriter-should-write-your-api-docs.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "API writing - just copy and paste?"
+date: "2017-06-27 06:34:02"
+image: blog-header/api-journey.jpg
+categories: api
+authors: ["Birgit"]
+---
+
+Can you imagine that?
+There are people who think that API writing is just about copying and pasting text.
+Or that API writers just make things nice and shiny; polish some wording here and there.
+I'm an API writer and I was super sad to hear that.
+When I addressed this with some of my colleagues, they said, of course they know that this is **not** what I do.
+But for them API documentation was a very abstract topic and they were very interested in what exactly I do.
+So I took them on a little API docs journey for our brand new software...
+
+## Speaking up for API writers
+
+If you're not born and raised as a developer, but still have to deal with APIs, you have to learn a lot, I mean A LOT.
+API writers need a thorough understanding of the API, its functionality, authentication, parameters, return types, what it supports and what not...
+
+They have to be flexible, proactive, sensitive, always present, well-organized...
```

Comment:
It's the third time "..."
maybe just put a full stop at the end of one of the three sentences?